config/database.yml

```diff
@@ -29,19 +29,19 @@ development:
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user running Rails.
-  #username: cojos
+  username: postgres
```

Comment:
most of the time in development you should not change this config, because you could affect the work of your peers.

In this case, looks like you need to add a role for your Postgres, this article maybe can help https://condef5.com/articles/install-and-use-postgres/ (Setup roles section)

Reply:
Oh!, I'm sorry. 
This file should not have been uploaded to the remote.
It happens that I have a somewhat strange installation of postgres on my machine (windows), that I need to modify the database file to run the migrations and the seeder.
Now I fix it. :)